src/js/base/module/Statusbar.js

```diff
@@ -37,5 +37,6 @@ export default class Statusbar {
 
   destroy() {
     this.$statusbar.off();
+    this.$statusbar.remove();
```

Comment:
This is removed by https://github.com/summernote/summernote/pull/2592.

Reply:
I didn't edit this either.